src/magellan_firmware/firmware/magellan_controller/Robot.cpp

```diff
@@ -4,13 +4,26 @@ Robot::Robot(ros::NodeHandle& nh) :
     current_state_(MODE_DISABLED),
     nh_(nh),
     transmitter_(nh),
-    heartbeat_() {
+    heartbeat_(),
+    throttle_pwm_(ESC_PWM),
+    steering_pwm_(SERVO_PWM) {
 }
 
 void Robot::TeleopInit() {
 }
 
 void Robot::TeleopPeriodic() {
+    // Get the throttle percent from the Transmitter Interface
+    double throttle_percent_ = transmitter_.throttle_percent();
+    
+    //Pass it to PWM
+    throttle_pwm_.Set(throttle_percent_);
+
+    // Get the steering angle from the Transmitter Interface
+    double steering_angle_ = transmitter_.steering_angle();
```

Comment:
Remember: The values that the `PWM` class expects to its `Set` method are -1 to +1, where 0 is middle/straight.

I believe `steering_angle()` returns values -90 to +90, so you'll want to do some scaling here (divide by 90). You can also see where this number is generated in `TransmitterInterface`.